tests/expected/function-contract/interior-mutability/transmute/cell.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Zfunction-contracts
+
+/// The objective of this test is to check the modification of a Cell used as interior mutability in an immutable struct
```

Comment:
This test does not need transmute and it is already covered by the other `cell.rs` test.